core/os/time.cpp

```diff
@@ -324,63 +324,61 @@ String Time::get_offset_string_from_offset_minutes(int64_t p_offset_minutes) con
 }
 
 Dictionary Time::get_datetime_dict_from_system(bool p_utc) const {
-	OS::Date date = OS::get_singleton()->get_date(p_utc);
-	OS::Time time = OS::get_singleton()->get_time(p_utc);
+	OS::DateTime dt = OS::get_singleton()->get_datetime(p_utc);
 	Dictionary datetime;
-	datetime[YEAR_KEY] = date.year;
-	datetime[MONTH_KEY] = (uint8_t)date.month;
-	datetime[DAY_KEY] = date.day;
-	datetime[WEEKDAY_KEY] = (uint8_t)date.weekday;
-	datetime[DST_KEY] = date.dst;
-	datetime[HOUR_KEY] = time.hour;
-	datetime[MINUTE_KEY] = time.minute;
-	datetime[SECOND_KEY] = time.second;
+	datetime[YEAR_KEY] = dt.year;
+	datetime[MONTH_KEY] = (uint8_t)dt.month;
+	datetime[DAY_KEY] = dt.day;
+	datetime[WEEKDAY_KEY] = (uint8_t)dt.weekday;
+	datetime[DST_KEY] = dt.dst;
+	datetime[HOUR_KEY] = dt.hour;
+	datetime[MINUTE_KEY] = dt.minute;
+	datetime[SECOND_KEY] = dt.second;
 	return datetime;
 }
 
 Dictionary Time::get_date_dict_from_system(bool p_utc) const {
-	OS::Date date = OS::get_singleton()->get_date(p_utc);
+	OS::DateTime dt = OS::get_singleton()->get_datetime(p_utc);
 	Dictionary date_dictionary;
-	date_dictionary[YEAR_KEY] = date.year;
-	date_dictionary[MONTH_KEY] = (uint8_t)date.month;
-	date_dictionary[DAY_KEY] = date.day;
-	date_dictionary[WEEKDAY_KEY] = (uint8_t)date.weekday;
-	date_dictionary[DST_KEY] = date.dst;
+	date_dictionary[YEAR_KEY] = dt.year;
+	date_dictionary[MONTH_KEY] = (uint8_t)dt.month;
+	date_dictionary[DAY_KEY] = dt.day;
+	date_dictionary[WEEKDAY_KEY] = (uint8_t)dt.weekday;
+	date_dictionary[DST_KEY] = dt.dst;
```

Comment:
Now that my [previous concern](https://github.com/godotengine/godot/issues/60327#issuecomment-1101453623) of discarding information is no longer relevant, I think we should remove DST from the date getter, and only include it in `get_datetime`.
```suggestion
```

Reply:
In v4, `get_datetime` isn't exposed to user anymore, all access to date informations was moved to `Time` class, so removing DST keys from `Time.get_X` will prevent user to any access to it

Reply:
Maybe I'm missing something here, but what I am saying is that we can include DST information in `Time::get_datetime_dict_from_system` and exclude it in `Time::get_date_dict_from_system`.

Reply:
My bad, you meant just removing the DST key in `get_date_dict_from_system`, right?

Reply:
Done! The [documentation](https://docs.godotengine.org/en/latest/classes/class_time.html#class-time-method-get-date-dict-from-system) should be updated

Reply:
Should I stash this 3 commits?

Reply:
Thanks, the documentation should be updated in this PR, in `doc/classes/Time.xml`.